dev_requirements.txt

```diff
@@ -3,18 +3,22 @@
 git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
 git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
 
+black==21.12b0
```

Comment:
Let's not use the beta version here. I've been updating that in other PRs to `22.3.0` instead